sphinx_gallery/tests/test_binder.py

```diff
@@ -115,9 +116,10 @@ def apptmp():
     assert url == expected
 
 
-def test_gen_binder_rst():
+def test_gen_binder_rst(tmpdir):
     """Check binder rst generated correctly."""
     gallery_conf_base = {'gallery_dirs': None, 'src_dir': 'blahblah'}
```

Comment:
I set `'gallery_dirs': None` to avoid copying files over when testing, but we can change this back to what it was before

Reply:
Pushed a commit to be more like how it should in actually building